app/database/models/videos.py

```diff
@@ -0,0 +1,52 @@
+from app.database.sqlalchemy_extension import db
+
+
+
+class VideoModel(db.Model):
+
+    # Specifying table
+    __tablename__ = "videos"
+    __table_args__ = {"extend_existing": True}
+
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.Text())
+    url = db.Column(db.Text())
+    title = db.Column(db.String(200))
+    published = db.Column(db.Integer)
+    source = db.Column(db.String(200))
+    channel = db.Column(db.String(200))
```

Comment:
100 is enough

Reply:
only for channel or for channel, title and source?